modules/kubernetes-addons/cert-manager-istio-csr/versions.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
```

Comment:
There are no resources used in this addon so all of the required providers can be removed and just leave the required terraform version